src/inventory/belt.py

```diff
@@ -25,7 +27,7 @@ def open(img: np.ndarray = None) -> np.ndarray:
 def close(img: np.ndarray = None) -> np.ndarray:
     img = grab() if img is None else img
     if not is_visible(ScreenObjects.BeltExpandable, img):
-        keyboard.send("esc")
+        keyboard.send(hotkeys.d2r_keymap[hotkeys.HotkeyName.OpenMenu])
```

Comment:
This place and others might be optimal to use new hotkey for ClearScreen. Though this may require some logic since I dont think ClearScreen is a required hotkey. Not necessarily in this PR so we can avoid regression, just thinking out loud.

Reply:
I was thinking about that too but that's why I opted to use OpenMenu since it's a hidden hotkey defaulted to be "Esc" (and is not actually bindable in-game); it may be worth it to have a convenience somehow where the logic uses ClearScreen if ClearScreen not None else OpenMenu.